Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM woorank/docker-node-babel
+EXPOSE 8080-8082
+
+ADD node_modules/ /node_modules
+ADD package.json /
+ADD src/ /src
+ADD crossdock/src /crossdock/src
+ADD .babelrc /
+
+RUN npm install tchannel
```

Comment:
to illustrate the point above, why do we need to re-install tchannel if we already copied all modules? Is it because tchannel (or its dependencies like thriftrw) uses native code?